CHANGELOG.md

```diff
@@ -10,6 +10,37 @@ Monitor the release status by regions at [AKS-Release-Tracker](https://releases.
 
 ### Release notes 
 
+* Features
+
+* Bug Fixes 
+  * Fix for preventing [cilium-operator](https://learn.microsoft.com/en-us/azure/aks/azure-cni-powered-by-cilium)  from restarting unmanaged coredns pods
+  * Fix for - [CVE-2023-44487](https://www.cve.org/CVERecord?id=CVE-2023-44487) - The HTTP/2 protocol allows a denial of service (server resource consumption) because request cancellation can reset many streams quickly
+  * Fix for AKS Not Honoring/ Returning [PrivateEndpointConnection](https://learn.microsoft.com/en-us/azure/aks/private-clusters?tabs=azure-portal#use-a-private-endpoint-connection) description field
+  * Fis for PUT on ManagedCluster allowing more than the [maximum tag limit](https://learn.microsoft.com/en-us/Azure/azure-resource-manager/management/tag-resources#limitations) of 50 in some rare cases
+  * Fix for Failure to create multiple agent pools concurrently when using the same PodSubnetID- [Dynamic IP Allocation mode](https://learn.microsoft.com/en-us/azure/aks/configure-azure-cni-dynamic-ip-allocation)
+
+* Behavioral Changes
+  * Disable creation of new nodepools with OSSKU cblmariner as CBL-Mariner 1.0 is end of life
```

Comment:
What is the suggested action to customers who are using cblmariner as os-sku right now? Is it "AzureLinux"?

Reply:
There are currently no existing doc for Mariner 1.0 (it was never advertised). There are 2 customers using a very small amount of it so adding it in a doc might cause confusion. PM suggested a release notes announcement should be fine or we can take it out entirely.